src/currentprocess/terminalsource.rs

```diff
@@ -73,14 +74,17 @@ enum TerminalInnerLocked {
 }
 
 impl ColorableTerminal {
-    /// Construct a terminal for the selected stream. Colors written to the
-    /// terminal will be discarded if the stream is not a tty.
+    /// Construct a terminal for the selected stream.
+    /// Colors written to the terminal will be discarded if the stream is not a tty,
+    /// unless the `RUSTUP_TERM_COLOR` override environment variable is set.
```

Comment:
```suggestion
    /// A terminal that supports colorisation of a stream.
    /// If `RUSTUP_TERM_COLOR` is set to `always`, or if the stream is a tty and
    /// `RUSTUP_TERM_COLOR` either unset or set to `auto`, then 
    /// color commands will be sent to the stream.
    /// Otherwise color commands are discarded.
```

Reply:
Done.